docs/UserGuide.md

```diff
@@ -91,34 +91,37 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com`
 * `add n/Betsy Crowe t/C++ e/betsycrowe@example.com p/1234567 t/Java a/Major=Data Science`
 
-### Listing all persons : `list`
+### Listing all persons: `list`
 
 Shows a list of all persons in the database.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a person: `edit`
 
 Edits an existing person in the database.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [t/TAG] [a/ATTRIBUTE_NAME=ATTRIBUTE_VALUE] [ra/ATTRIBUTE_NAME] …​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [t/TAG]… [a/ATTRIBUTE_NAME=ATTRIBUTE_VALUE]… [ra/ATTRIBUTE_NAME]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+
+For tags:
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `t/` without specifying any tags after it.
+
+For attributes:
 * Update of attributes must specify both the attribute name and attribute value.
 * Update of attributes will update an existing attribute if it exists, else it will add it as a new attribute.
 * Removal of attributes only requires specifying the attribute name.
-* Regardless of the order that update attribute and remove attribute parameters are provided in,
-* All attribute updates will be processed and executed first, before attribue removal.
+* Regardless of the order that update attribute and remove attribute parameters are provided in, all attribute updates will be processed and executed first, before attribute removal.
 
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-*  `edit 1 a/graduation year=2027` Adds or edits an attribute named "Graduation Year" to the 1st person.
-*  `edit 1 ra/graduation year` Removes the attribute named "Graduation Year" from the 1st person.
+*  `edit 1 a/Graduation Year=2027` Adds or edits an attribute named "Graduation Year" to the 1st person.
```

Comment:
We might add that the user cannot use /, \, or = in the names and values. The same goes to the filter command (sorry).